dttools/src/copy_tree.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2013- The University of Notre Dame
```

Comment:
Accept. 
Oops. I directly copied these copyright info from other source files and got tricked.